<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>543</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>543</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-18</x>
      <y>0</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>24</x>
            <y>29</y>
            <width>250</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButtonConnect">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDisconnect">
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>23</x>
            <y>-4</y>
            <width>251</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_20">
           <item>
            <widget class="QLineEdit" name="lineEditIPServidor">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Digite o IP do Servidor</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>120</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QSlider" name="horizontalSliderMin">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumberMin"/>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>104</y>
            <width>37</width>
            <height>12</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>174</y>
            <width>37</width>
            <height>12</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>190</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QSlider" name="horizontalSliderMax">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumberMax"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>250</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Timing(s)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderTiming">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTiming">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_6">
          <property name="geometry">
           <rect>
            <x>24</x>
            <y>319</y>
            <width>250</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QPushButton" name="pushButtonStart">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonStop">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="labelConnected">
          <property name="geometry">
           <rect>
            <x>28</x>
            <y>60</y>
            <width>241</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: red;</string>
          </property>
          <property name="text">
           <string>Disconnected</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QTextBrowser" name="textBrowserLog">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>271</width>
            <height>351</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
